tests/js/all-tests.js

```diff
@@ -1,4 +1,9 @@
 // Run all tests in this package in series.
 "use strict";
+
+var events = require("events");
+events.EventEmitter.defaultMaxListeners = 100;
```

Comment:
It's a variation on the behavior described here:

http://www.jongleberry.com/understanding-possible-eventemitter-leaks.html

Node itself complains if you fail to clean up more than 10 listeners in a given run.